Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -273,6 +273,42 @@ open class RouteController: NSObject {
         }
         // Check for faster route proactively (if reroutesProactively is enabled)
         refreshAndCheckForFasterRoute(from: location, routeProgress: routeProgress)
+        
+        verifyNavigationTilesVersion()
+    }
+    
+    private var wantsSwitchTilesVersion = false
```

Comment:
nit: I know it is not the patter followed in the SDK but it is recommended to specify the type of the variable if it is in a top level declaration/class/struct/etc. It makes compiler do less work.